src/components/container/IntradayHeartRateChart/IntradayHeartRateChart.data.ts

```diff
@@ -0,0 +1,44 @@
+import { add, addMinutes, startOfDay } from "date-fns";
+import { DeviceDataV2Aggregate } from "@careevolution/mydatahelps-js";
+export type IntradayHeartRateData = { [key: string]: DeviceDataV2Aggregate };
```

Comment:
This should be: `import { IntradayHeartRateData } from '../../../helpers/heart-rate-data-providers/combined-intraday-heart-rate-providers'`.

The type definition here is out of date (using a `string` key).